haas/cli.py

```diff
@@ -718,7 +718,7 @@ def show_node(node):
 
     q = check_clientlib_response(lambda: C.node.show(node))
     for item in q.items():
-        sys.stdout.write("%s\t  :  %s\n" % (item[0], item[1]))
+        print item
```

Comment:
seems like most of  cli is using `sys.stdout.write` why switching only this one to `print`?